docs/api/request.md

```diff
@@ -117,10 +117,10 @@ this.request.charset
 
   For example "color=blue&size=small":
 
-```js
+```json
 {
-  color: 'blue',
-  size: 'small'
+  "color": "blue",
+  "size": "small"
```

Comment:
Unfortunately, this is not valid JS code:

```
{
  color: 'blue',
  size: 'small'
}
```

It is closer to JSON, but then it misses the quotation marks.